kfp/kfp_support_lib/kfp_v2_workflow_support/Makefile

```diff
@@ -0,0 +1,78 @@
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../..
+include ${REPOROOT}/.make.versions
+include ${REPOROOT}/kfp/requirements.env
+
+# Include the common rules.
+# Use "make help" to see them.
+include ${REPOROOT}/.make.defaults
+
+# Command to run pytest
+PYTHON_VERSION=$(shell $(PYTHON) --version)
+VENV_ACTIVATE=venv/bin/activate
+
+DEPLOY_KUBEFLOW ?= 0
+
+clean::
+	@# Help: Clean up the distribution build and the venv
+	rm -r dist venv || true
+	rm -rf src/*egg-info || true
+	rm -rf *.back || true
+
+
+.check-env:: .check_python_version
+	@echo "Checks passed"
+
+set-versions:: .check-env
+	@# Help: Copy the Makefile distribution version into the pyproject.toml
+	sed -i.back 's/^version[ ]*=.*/version = "'${DPK_LIB_KFP_VERSION_v2}'"/' pyproject.toml
+	sed -i.back 's/data-prep-toolkit==[0-9].*/data-prep-toolkit==${DPK_LIB_VERSION}",/' pyproject.toml
+	sed -i.back 's/kfp==[0-9].*/kfp==${KFP_v2}",/' pyproject.toml
+	sed -i.back 's/ray=[0-9].*/ray==${RAY}",/' pyproject.toml
+
+build:: set-versions venv
+ifeq ($(KFPv2), 0)
+	echo "Skipping build as KFPv2 is not defined"
+else
```

Comment:
As I wrote the upper Makefile executes recursively `make build` or `make test` in ALL sub-folders, so it will be called in kfp_v2_workflow_support even if we build for KFPv1 and visa versa.